script.js

```diff
@@ -1,4 +1,275 @@
+  }
+
+  //Check if player 2 has rolled, after player 1 has rolled
+  if (hasPlayer1Rolled == 1 && hasPlayer2Rolled == 1) {
+    return `Welcome Player 2. <br><br>You rolled ${player2DiceRolls[0]} for Dice 1 and ${player2DiceRolls[1]} for Dice 2. <br><br>Choose the order of the dice. <br>Input "dice 1" if you wish for Dice 1's number to be in front of Dice 2's number. <br>Input "dice 2" if you wish for Dice 2's number to be in front of Dice 1's number.`;
+  }
+};
+
+//Create a function that outputs the notification statement after the players' have chosen which dice to concatenate and return the concatenated number
+var statementForConcatenation = function (userInput) {
+  //Check if only player 1 has rolled
+  if (hasPlayer1Rolled == 1 && hasPlayer2Rolled == 0) {
+    return `Player 1, you chose ${capitalizer(
+      userInput
+    )} first. <br>Your number is ${player1ConcatenatedNumber}. <br>It is now Player 2's turn.`;
+  }
+
+  //Check if player 2 has rolled, after player 1 has rolled
+  if (hasPlayer1Rolled == 1 && hasPlayer2Rolled == 1) {
+    return `Player 2, you chose ${capitalizer(
+      userInput
+    )} first. <br>Your number is ${player2ConcatenatedNumber}. <br>It is now time to see who won! Type in "compare" to see who won.`;
+  }
+};
+
+//Create a function that compares the two numbers and output a who wins statement.
+var comparingWinnerStatement = function () {
+  //Create a variable that is a statement to let the players know how to reset the game
+  var howToReset = `Please input "reset" if you wish to reset the game and play again.`;
+
+  //Check if there is a tie
+  if (player1ConcatenatedNumber == player2ConcatenatedNumber) {
+    return `Both of your numbers are ${player1ConcatenatedNumber}. <br><br>It's a tie! <br><br>${howToReset}`;
+  }
+
+  //Check if player 1 won
+  if (player1ConcatenatedNumber > player2ConcatenatedNumber) {
+    return `Player 1's number is ${player1ConcatenatedNumber}. <br>Player 2's number is ${player2ConcatenatedNumber}. <br><br>Player 1 won! <br><br>${howToReset}`;
+  }
+
+  //Check if player 2 won
+  if (player2ConcatenatedNumber > player1ConcatenatedNumber) {
+    return `Player 1's number is ${player1ConcatenatedNumber}. <br>Player 2's number is ${player2ConcatenatedNumber}. <br><br>Player 2 won! <br><br>${howToReset}`;
+  }
+};
+
+//Create a function that resets the game
+var gameReset = function () {
+  hasPlayer1Rolled = 0;
+  hasPlayer2Rolled = 0;
+  player1DiceRolls = [];
+  player2DiceRolls = [];
+  player1ConcatenatedNumber = 0;
+  player2ConcatenatedNumber = 0;
+  timeToReset = 0;
+  return `The game has been reset! You may play again by pressing the "Submit" button.`;
+};
+
+//Create a function that receives the user input and returns the correct output statements
+var beatThatGame = function (userInput) {
+  //Make the input entirely lowercase
+  userInput = userInput.toLowerCase();
```

Comment:
good standardising of user input regardless of case, nice that you're thinking about the user experience!